pkg/php/php.go

```diff
@@ -41,6 +42,8 @@ const (
 	dependencyHashKey = "dependency_hash"
 
 	composerVersionKey = "php"
+	// composerArgsEnv is an environment variable used to pass custom composer variables.
+	composerArgsEnv = "COMPOSER_ARGS"
```

Comment:
Let's also move the new variable to gcp_buildpacks/pkg/env/env.go where we store all the env vars.